tests/assets/recipes/parametric-daylight-factor/example-arguments.yaml

```diff
@@ -0,0 +1,13 @@
+artifacts:
+  - name: models-folder
+    source:
+      type: project-folder
+      path: path/to/models
+parameters:
+  - name: model-names
+    value: |
```

Comment:
fancy!

Reply:
Meh... Does the trick. I'd be happier with actual JSON but we'll get there later :)